data-prepper-plugins/map-values-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/mapvalues/MapValues.java

```diff
@@ -0,0 +1,179 @@
+
+
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.processor.mapvalues;
+
+import org.apache.commons.lang3.Range;
+import org.apache.commons.lang3.math.NumberUtils;
+import org.opensearch.dataprepper.expression.ExpressionEvaluator;
+import org.opensearch.dataprepper.metrics.PluginMetrics;
+import org.opensearch.dataprepper.model.annotations.DataPrepperPlugin;
+import org.opensearch.dataprepper.model.annotations.DataPrepperPluginConstructor;
+import org.opensearch.dataprepper.model.event.Event;
+import org.opensearch.dataprepper.model.processor.AbstractProcessor;
+import org.opensearch.dataprepper.model.processor.Processor;
+import org.opensearch.dataprepper.model.record.Record;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Collection;
+import java.util.Map;
+import java.util.LinkedHashMap;
+import java.util.HashMap;
+import java.util.Objects;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static org.opensearch.dataprepper.logging.DataPrepperMarkers.EVENT;
+
+
+@DataPrepperPlugin(name = "map-values", pluginType = Processor.class, pluginConfigurationType = MapValuesConfig.class)
```

Comment:
Looks like we are mostly using "_" and not "-" in the name, please change the name to "map_values"

Reply:
ack.